shoop/admin/modules/products/views/edit.py

```diff
@@ -161,3 +161,17 @@ def form_valid(self, form):
 
     def get_toolbar(self):
         return EditProductToolbar(view=self)
+
+    def get_context_data(self, **kwargs):
+        context = super(ProductEditView, self).get_context_data(**kwargs)
+        orderability_errors = []
+        if self.object.pk:
```

Comment:
That said, I think `self.request.shop` isn't a valid thing to consider in an admin context.